src/neo4j_graphrag/experimental/components/entity_relation_extractor.py

```diff
@@ -100,28 +102,19 @@ def balance_curly_braces(json_string: str) -> str:
     return "".join(fixed_json)
 
 
-def fix_invalid_json(invalid_json_string: str) -> str:
-    # Fix missing quotes around field names
-    invalid_json_string = re.sub(
-        r"([{,]\s*)(\w+)(\s*:)", r'\1"\2"\3', invalid_json_string
-    )
-
-    # Fix missing quotes around string values, correctly ignoring null, true, false, and numeric values
-    invalid_json_string = re.sub(
-        r"(?<=:\s)(?!(null|true|false|\d+\.?\d*))([a-zA-Z_][a-zA-Z0-9_]*)\s*(?=[,}])",
-        r'"\2"',
-        invalid_json_string,
-    )
-
-    # Correct the specific issue: remove trailing commas within arrays or objects before closing braces or brackets
-    invalid_json_string = re.sub(r",\s*(?=[}\]])", "", invalid_json_string)
```

Comment:
Given we're replacing the old JSON fixing code here does json_repair completely cover the situations these regexes were meant to fix? Would there be any reason to keep these around?

Reply:
Yes there were previous unit tests that still pass 👍 

When I combined both json_repair and the old fixing code, it causes more problems. I think this was because the original code to fix the json string was not perfect and lacked many cases. So IMO it's better to just rely on json_repair